src/tests/0000ztrips.js

```diff
@@ -410,7 +408,7 @@ describe('trips tests', () => {
       });
   });
   it('should return the number of trips created by a user', (done) => {
-    const startDate = '2020-01-30';
+    const startDate = '2020-02-04';
```

Comment:
@jo-rdan I guess this was the reason why this test was failing, trips were getting created but couldn't find those created before 2020-01-30, now that you have changed to today's date, it is passing but starting tomorrow going forward they might fail. Instead of hardcoding the startDate, could you use ones that are created during this test suites.

Reply:
Before you make the above changes, change the startDate to yesterday or a week ago and run the test.